examples/README.md

```diff
@@ -0,0 +1,24 @@
+<!--
+    Copyright 2014 The Gfx-rs Developers.
```

Comment:
I never know about these... do you want me to change them all? The others are 2014, atleast the one I copy/pasted from